src/test/java/htsjdk/variant/variantcontext/writer/VariantContextWriterBuilderUnitTest.java

```diff
@@ -347,6 +347,25 @@ public void testIndexingOnTheFly() {
     }
   }
 
+  @Test
+  public void testIndexingOnTheFlyForPathStream() throws IOException {
+    final VariantContextWriterBuilder builder = new VariantContextWriterBuilder()
```

Comment:
Indent should be 4 spaces, not 2. I realize the rest of the file is using 2, but IMO we should move to a common convention for all files, starting with new/updated code.

Reply:
I can reindent the file pretty easily if you would like? It seems orthogonal to this branch however.

Reply:
I would just fix the lines that you're adding/modifying.